routes/index.js

```diff
@@ -5,21 +5,20 @@ var request = require('request');
 /* GET home page. */
 router.get('/', function(req, res, next) {
   signalId = Math.floor(Math.random() * (5823 - 1) + 1);
+  console.log("request: 'http://www.acessibilidadebrasil.org.br/libras_3/ajax/getWordById/" + signalId + "'");
```

Comment:
Is it still needed?

Reply:
No, I saw after and already remove it! =)